### PR TITLE
Implemented tests for Content View association via API

### LINKED ISSUE #687: Write tests for Content View association via API
Please write the tests for the following (already written) stubs:

* test_associate_view_rh
* test_associate_view_rh_custom_spin
* test_associate_view_custom_content
* test_cv_associate_puppet_repo_negative
* test_cv_associate_components_composite_negative
* test_cv_associate_composite_dupe_repos_negative
* test_cv_associate_composite_dupe_modules_negative

### PR DESCRIPTION
**Pull request by oshtaier**
Closes #687

### CODE REVIEW

**Comment by oshtaier** [comment]
```
nosetests tests/foreman/api/test_contentview.py -m test_cv_associate_custom_content
.
----------------------------------------------------------------------
Ran 1 test in 37.881s

OK

nosetests tests/foreman/api/test_contentview.py -m test_cv_associate_puppet_repo_negative
.
----------------------------------------------------------------------
Ran 1 test in 22.746s

OK

nosetests tests/foreman/api/test_contentview.py -m test_cv_associate_composite_dupe_repos_negative
.
----------------------------------------------------------------------
Ran 1 test in 35.038s

OK

nosetests tests/foreman/api/test_contentview.py -m test_cv_associate_composite_dupe_modules_negative
.
----------------------------------------------------------------------
Ran 1 test in 39.294s

OK

nosetests tests/foreman/api/test_contentview.py -m test_cv_associate_components_composite_negative
.
----------------------------------------------------------------------
Ran 1 test in 87.442s

OK

nosetests tests/foreman/api/test_contentview.py -m test_cv_associate_rh
..
----------------------------------------------------------------------
Ran 2 tests in 112.305s

OK
```

**Comment by oshtaier** [comment]
Please, do not merge until https://github.com/SatelliteQE/nailgun/pull/71 is merged as tests will fail without necessary changes in naligun

File: robottelo/common/constants.py [hunk 1]
```diff
@@ -144,6 +144,7 @@
 REPOSET = {
     'rhct6': "Red Hat CloudForms Tools for RHEL 6 (RPMs)",
     'rhel6': "Red Hat Enterprise Linux 6 Server (RPMs)",
+    'rhelc6': "Red Hat Enterprise Linux 6 Server - RH Common (RPMs)",
```
Comment (sthirugn) [inline robottelo/common/constants.py]: to make it more readable, s/'rhelc6'/'rhel6common'?

Comment (sthirugn) [inline robottelo/common/constants.py]: never mind, all other repos are using this same format, we need to improve this later.

**Comment by elyezer** [comment]
ACK pending https://github.com/SatelliteQE/nailgun/pull/71 and new nailgun version.

**Comment by Ichimonji10** [comment]
Please take a look at SatelliteQE/nailgun#72. That pull request contains some important bug fixes, so I'm aiming for a new release right after that is merged.

File: robottelo/common/constants.py (hunk 1 above)
Comment (omaciel) [inline robottelo/common/constants.py]: Eventually we will need to start using ``Red Hat Satellite Tools 6 Beta (for RHEL 6 Server) (RPMs)`` as the **RH Common** channel will become 'obsolete'

**Comment by Ichimonji10** [comment]
ACK pending NailGun 0.14.0 being used by Robottelo.

File: robottelo/common/constants.py (hunk 1 above)
Comment (elyezer) [inline robottelo/common/constants.py]: Seems that we need an issue for this. will create it.

Comment (elyezer) [inline robottelo/common/constants.py]: Done #2244.

Comment (Ichimonji10) [inline robottelo/common/constants.py]: :+1:

**Comment by Ichimonji10** [comment]
Depends on #2245.

**Pull request merged**